From 9a928260777e761c0a1f01878a5336a44f8dd8a1 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Fri, 22 Nov 2024 15:28:09 -0800
Subject: [PATCH 2/2] default fat lto objects

---
 mkspecs/common/gcc-base.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mkspecs/common/gcc-base.conf b/mkspecs/common/gcc-base.conf
index ae583262..bbff7e33 100644
--- a/mkspecs/common/gcc-base.conf
+++ b/mkspecs/common/gcc-base.conf
@@ -50,7 +50,7 @@ QMAKE_CFLAGS_YACC          += -Wno-unused -Wno-parentheses
 QMAKE_CFLAGS_HIDESYMS      += -fvisibility=hidden
 QMAKE_CFLAGS_EXCEPTIONS_OFF += -fno-exceptions
 QMAKE_CFLAGS_SPLIT_SECTIONS += -ffunction-sections -fdata-sections
-QMAKE_CFLAGS_LTCG           = -flto -fno-fat-lto-objects
+QMAKE_CFLAGS_LTCG           = -flto -ffat-lto-objects
 QMAKE_CFLAGS_LTCG_FATOBJECTS = -ffat-lto-objects
 QMAKE_CFLAGS_DISABLE_LTCG   = -fno-lto
 QMAKE_CFLAGS_SHSTK          = -mshstk
-- 
2.47.1

